scipy/stats/_distn_infrastructure.py

```diff
@@ -2493,7 +2585,7 @@ def fit(self, data, *args, **kwds):
         # instead of an optimizer, but sometimes no solution exists,
         # especially when the user fixes parameters. Minimizing the sum
         # of squares of the error generalizes to these cases.
-        vals = optimizer(func, x0, args=(ravel(data),), disp=0)
```

Comment:
`ravel` uncensored arrays above. Add a note that we're doing it for backwards compatibility.

Reply:
Done.